features/steps/document.py

```diff
@@ -162,6 +167,14 @@ def when_I_change_the_new_section_layout_to_landscape(context):
 
 # then ====================================================
 
+@then('document.bookmarks is a Bookmarks object')
+def then_document_bookmarks_is_a_list_containing_three_bookmarks(context):
+    bookmarks = context.document.bookmarks
+    actual = bookmarks.__class__.__name__
+    expected = 'Bookmarks'
+    assert actual == expected, 'bookmarks is a %s object' % actual
```

Comment:
While it may seem a trivial test, all we want to do here is establish we got a Bookmarks object on this call. Any particular behaviors of that bookmarks object is for another test. These "..is an X object" tests are quite effective at driving (and defending) the top-level behaviors they're designed to test.